Jenkinsfile

```diff
@@ -1,47 +1,103 @@
 #!groovy
 @Library('waluigi@release/7') _
 
-standardProperties()
```

Comment:
Was this deleted by accident? I think we need to put it back

Reply:
Yes, removed while testing only